### ISSUE TITLE
PAGE XML contains `TextEquiv` with empty `Unicode`

### ISSUE THREAD

**Issue by stweil** [UB-Mannheim/reichsanzeiger-gt#1]
The PAGE XML files contain lots of text regions without text in their `TextEquiv` and a few text files without text in their `TextEquiv`:

```
# Text regions without text.
% git grep "^                <Unicode></Unicode>" | wc -l
   17097
# Text lines without text:
% git grep "^                    <Unicode></Unicode>" | wc -l  
       6
```

Text from regions with text in lines but without text in the region gets lost when the PAGE XML file is converted to pure text using `ocr-transform`.

**Comment by stweil** [comment]
https://github.com/UB-Mannheim/Fibeln also has 11 files which contain text regions without text. Those files where also created using Transkribus. This indicates that it might be a general problem of that software.

**Comment by tsmdt** [comment]
Solved: [a65f5ba91c1ea1be13f9443a96cedbcb249b346d](https://github.com/UB-Mannheim/reichsanzeiger-gt/commit/a65f5ba91c1ea1be13f9443a96cedbcb249b346d)
